### PR TITLE
Improve components

### PR DESCRIPTION
**Pull request by frozenhelium**
- Add content view type in Container
- Add option to expand image when clicked
- Fix table overflow
- Reduce map height to 60vh in mobile views
- Add approval workflow for the DREF

### CODE REVIEW

File: src/components/ChartAxisX/styles.module.css [hunk 1]
```diff
@@ -1,6 +1,7 @@
 .chart-axis-x {
     .x-axis-tick-text {
         transition: var(--go-ui-duration-transition-medium) ease-in-out transform;
+        background-color: blue;
```
Comment (tnagorra) [inline src/components/ChartAxisX/styles.module.css]: We should use a design token?

File: src/components/Table/index.tsx [hunk 2]
```diff
@@ -168,7 +166,9 @@ function Table<D, K extends string | number, C extends Column<D, K, any, any>>(
         sum(columns.map((c) => columnWidths[c.id]))
     ), [columnWidths, columns]);
 
-    const isEmpty = isNotDefined(data) || data.length === 0;
+    const isEmpty = isNotDefined(data)
+        || data.length === 0
+        || Object.keys(columnWidths).length === 0;
```
Comment (tnagorra) [inline src/components/Table/index.tsx:171]: Let's add NOTE why we are not showing information if we do not have columnWidths

Comment (frozenhelium) [reply src/components/Table/index.tsx:171]: I've no idea, it was already there. I just moved the lines!

File: src/views/AccountDrefApplications/DrefTableActions/index.tsx [hunk 3]
```diff
@@ -145,18 +244,49 @@ function DrefTableActions(props: Props) {
...
+        ],
+    );
+
+    const handleDrefAllocationExport = useCallback(
+        () => {
+            if (applicationType === 'DREF') {
+                fetchDref(id);
+            } else if (applicationType === 'OPS_UPDATE') {
+                fetchOpsUpdate(id);
+            }
```
Comment (tnagorra) [inline src/views/AccountDrefApplications/DrefTableActions/index.tsx:271]: What about Final Report?

File: src/views/AccountDrefApplications/DrefTableActions/index.tsx [hunk 4]
```diff
@@ -165,8 +295,11 @@ function DrefTableActions(props: Props) {
                     {canApprove && (
                         <DropdownMenuItem
                             name={undefined}
-                            type="button"
+                            type="confirm-button"
                             icons={<CheckLineIcon className={styles.icon} />}
+                            confirmMessage="You're about to Approve this DREF. Once approved, it can no longer be edited. Are you sure, you want to Approve?"
```
Comment (tnagorra) [inline src/views/AccountDrefApplications/DrefTableActions/index.tsx:300]: Let's add translations

**Pull request merged**